numpy/core/src/multiarray/lowlevel_strided_loops.c.src

```diff
@@ -1823,3 +1825,193 @@ mapiter_@name@(PyArrayMapIterObject *mit)
 }
 
 /**end repeat**/
+
+
+
+#define ptr_assignment1(ptr1, val1, stride1) *ptr1 = val1; ptr1 += stride1; 
+#define ptr_assignment2(ptr1, val1, stride1, ptr2, val2, stride2) ptr_assignment1(ptr1, val1, stride1) *ptr2 = val2; ptr2 += stride2; 
```

Comment:
To be honest, I can't parse well "ptr_assignment2" means exactly.  The first macro seems OK to just inline, the second probably as well?